hawc/main/settings/unittest.py

```diff
@@ -13,7 +13,7 @@
 HAWC_FLAVOR = "PRIME"
 ANYONE_CAN_CREATE_ASSESSMENTS = True
 
-DATABASES["default"]["NAME"] = "hawc-fixture-test"
+DATABASES["default"]["NAME"] = "hawc-fixture"
```

Comment:
should we delete this line?  its' confusing b/c we specify the test-db name in base.py now, and our local settings now use this hawc-fixture line

Reply:
We use the `unittest` setting when we play around with our fixture database. Do you think that should be relegated entirely to our `local` setting, and change our documentation to use `local` when playing with the fixture database?

Also we can move the test db declaration here instead instead of base, if you think that'd be a better place for it, since we don't _need_ it in base but its required here.

Reply:
based on chat w/ @rabstejnek; add `"TEST": {"NAME": "hawc-test"}` to `DATABASES` and a comment to describe what these two things mean, just b/c this is confusing for a new hawc user